terraform/app_service.tf

```diff
@@ -0,0 +1,80 @@
+resource "azurerm_service_plan" "main" {
+  name                = "ASP-CDT-PUB-VIP-CALITP-P-001"
+  location            = data.azurerm_resource_group.prod.location
+  resource_group_name = data.azurerm_resource_group.prod.name
+  os_type             = "Linux"
+  sku_name            = "P2v2"
+
+  lifecycle {
+    ignore_changes = [tags]
+  }
+}
+
+# app_settings, sticky_settings, and storage_account are managed manually through the portal since they contain secrets
```

Comment:
Possible we could (at least partially) move these into Terraform, but I didn't want to spend a ton of time on it.